core/digital-twin-lib/src/index.ts

```diff
@@ -37,6 +37,7 @@ import {
 import { SearchService } from './SearchService';
 import PermissionUtils from './utils/PermissionUtils';
 import SharingUtils from './utils/SharingUtils';
+import { Permissions, PermissionsContainer } from './interfaces/Permissions';
```

Comment:
Can we add `Interface` to all interfaces? I already was confused with the `DigitalTwin`, imported from the digital-twin-lib into the assets. :D

Reply:
Hm, I'm not quite sure yet what the best practice is in TS. Think in Angular they omit it usually. Will research it and refactor accordingly :)

Reply:
just my 2 cents:

As I understood it a class acts in TS more or less the same as an interface. Though if we have a well typed class there should be no need for an interface. Also we must not have a class with the same name as an interface. 
Both is indicated by the capital letter of the beginning.
Nevertheless the "Interface" suffix may help at the current state, but we should really resolve ambiguous naming in this case.